packages/proxy/lib/http/request-middleware.ts

```diff
@@ -185,6 +185,24 @@ const CalculateCredentialLevelIfApplicable: RequestMiddleware = function () {
   this.next()
 }
 
+const FormatCookiesIfApplicable: RequestMiddleware = function () {
+  if (this.req.headers['x-cypress-is-webdriver-bidi'] && this.req.headers.cookie) {
+    const cookies = this.req.headers.cookie
+    // in the case of BiDi, cookies come in as foo=bar;bar=baz and not foo=bar; bar=baz,
+    // i.e. they are delimited differently, which impacts some of our tests and our cookie splicing.
+    // this regex is to help make sure the cookies are fed in consistently
```

Comment:
Is this spelled out in a spec somewhere? Just curious if we might need to know if this might change in the future. Do we have any firefox tests that would break in that case?

Reply:
in the Bidi spec or the cookie spec. I don't think they are in either. If it does change my guess is it wouldn't have an impact since we are only checking for the deviation. but that would mean the code would effectively be dead